### PR TITLE
[LTC] Implement Lidless Gaze

### PR DESCRIPTION
**Pull request by Susucre**

### CODE REVIEW

**Comment by Susucre** [comment]
[[Lidless Gaze]]

File: Mage.Sets/src/mage/cards/l/LidlessGaze.java [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+            if (player != null) {
+                Card card = player.getLibrary().removeFromTop(game);
+                if (card != null) {
+                    controller.moveCardsToExile(card, source, game, true, exileId, exileName);
+                    cards.add(card);
+                }
+            }
+        }
+
+        for (Card card : cards.getCards(game)) {
```
Comment (Susucre) [inline Mage.Sets/src/mage/cards/l/LidlessGaze.java:83]: need a retainZone for the cards?

Comment (xenohedron) [reply Mage.Sets/src/mage/cards/l/LidlessGaze.java:83]: Seems fine as is.

**Review by xenohedron — APPROVED** [review]

File: Mage.Sets/src/mage/cards/l/LidlessGaze.java (hunk 1 above)
Comment (xenohedron) [reply Mage.Sets/src/mage/cards/l/LidlessGaze.java:83]: Okay, well, see my comment on the other PR with a similar question. I guess it doesn't hurt but I'm not sure what situation it guards against. Misthollow Griffin could potentially be a problem but I don't think retainZone helps with it.

**Pull request merged**